src/view/renderer.js

```diff
@@ -531,6 +533,25 @@ export default class Renderer {
 		}
 	}
 
+	/**
+	 * Marks all text nodes in a tree starting from passed element to be synced.
+	 *
+	 * @private
+	 * @param {module:engine/view/element~Element} viewElement View element whose all 'text' children should
+	 * be marked to sync. If element is 'text' itself it will be marked too.
+	 */
+	_markDescendantTextToSync( viewElement ) {
+		if ( viewElement ) {
+			if ( viewElement.is( 'text' ) ) {
+				this.markedTexts.add( viewElement );
+			} else if ( viewElement.is( 'element' ) ) {
+				for ( const child of viewElement.getChildren() ) {
+					this._markDescendantTextToSync( child );
+				}
+			}
```

Comment:
I wonder when this else path is taken (what else could `viewElement` be), but CC is 100%.

Reply:
Theoretically it shouldn't be anything else so simple `if - else` should be sufficient. But checking it in both case is more future proof.